src/pages/academy/index.tsx

```diff
@@ -86,14 +86,14 @@ const DerivBlog = ({ data }: DerivBlogProps) => {
 
     // We need to include the !is_uk_country check together with is_eu_country because 'gb'
     // is a valid country code for both EU and UK in our country base.
-    if (is_eu_country && !is_uk_country) {
+    if (is_eu && !is_uk) {
```

Comment:
For both `EU` and `UK` please use `is_eu_uk` instead

Reply:
I think we only need `is_eu` here. `!is_uk` not needed.

Reply:
Agree with Mohammad